build-utils/run_ci.sh

```diff
@@ -22,6 +22,11 @@ if [[ -z $COMMAND || -z $STUDY_KEY || -z $BRANCH || ($COMMAND == "deploy" && -z
   echo "        Deploy saved build of corresponding to given branch to specified TARGET_ENV"
   echo "    run-tests"
   echo "        Run tests for given study and given branch"
+  echo "    run-e2e-tests"
+  echo "        Run Playwright E2E tests for given branch against specified TARGET_ENV (dev, test or staging)"
+  echo "        STUDY_KEY is unused. CI API call will run all tests"
+  echo "        Example: ./run_ci.sh run-e2e-tests singular pepper-227-test dev"
```

Comment:
Seems like a branch name is unused as well. 
Is not it?

Reply:
It is used. `run_ci.sh` feeds `branch` value to CircleCI V2 API `/pipeline` request. `branch` is where the pipeline run against.
https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline

Reply:
I mean a branch name is required by the common script/pipeline request.
However it does not matter what branch is provided for e2e test launch.
Is it correct?